ui/pages/onboarding-flow/secure-your-wallet/index.scss

```diff
@@ -0,0 +1,42 @@
+.secure-your-wallet {
```

Comment:
This file could use a tiny bit of cleanup.  For example, I think we like an empty line between selectors.